README.md

```diff
@@ -1,4 +1,21 @@
-# carpenter
+# Arcaflow Plugin Image Builder
+
+Arcaflow Plugin Image Builder is a tool which has been developed for automatically test, build and publish Arcaflow plugins.
```

Comment:
automatically testing, building, and publishing Arcaflow plugins.

Reply:
thanks